ui/insight/rollup.config.mjs

```diff
@@ -3,8 +3,11 @@ import rollupProject from '@build/rollupProject';
 export default rollupProject({
   main: {
     name: 'LichessInsight',
-    input: 'src/main.js',
+    input: 'src/main.ts',
     output: 'insight',
-    js: true,
+    external: ['highcharts'],
+    globals: {
+      highcharts: 'Highcharts',
+    },
```

Comment:
This prevents rollup from bundling highcharts so that we don't send a duplicate copy to the client. Related changes in ui/@build/rollupProject/index.js.

I need this because I imported highcharts as a module. Highcharts isn't a regular module; it's a global UMD module. Typescript doesn't like this truth and will loudly error if I treat highcharts as a global. This error does not prevent the build from succeeding though, so an alternative to these additional rollup options is to import Highcharts as a global and either ignore typescript's error or find a way to silence it.